packages/extension-browser/src/devtools/views/controls/print-link.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import * as styles from './print-link.css';
+
+const onPrintHandler = () => {
+    window.print();
+};
+
+const PrintLink = () => {
+    return (
+        <span className={styles.root} onClick={onPrintHandler}>
+            Print
```

Comment:
This needs to be localized.
To do so, you need to add a new entry to the file `src/locales/en/messages.json` (e.g. `print`)
and then use the method `getMessage` (e.g. getMessage('print')).

Reply:
I didn't get the getMessage part, where do I need to add it? I'll add the part in the message.json

Reply:
We have an i18n util in `src/devtools/utils/i18n.ts` that export a method `getMessage`, this method will take of the localization. The idea is replace you hardcoded `Print` for a `{getMessage('print')}`

You need to add a new entry to the file `src/_locales/en/messages.json` with you new string e.g.
```json
"print": {
    "message": "Print",
    "description": "Label for button to print the result."
}
```

You can find an example on how we use it in the file `src/devtools/views/pages/results/header.tsx`.